tests/components/here_travel_time/test_sensor.py

```diff
@@ -171,26 +69,41 @@ async def test_car(hass, requests_mock_car_disabled_response):
             "destination_latitude": CAR_DESTINATION_LATITUDE,
             "destination_longitude": CAR_DESTINATION_LONGITUDE,
             "api_key": API_KEY,
+            "traffic_mode": traffic_mode,
+            "unit_system": unit_system,
+            "mode": mode,
         }
     }
     assert await async_setup_component(hass, DOMAIN, config)
     await hass.async_block_till_done()
-
     hass.bus.async_fire(EVENT_HOMEASSISTANT_START)
     await hass.async_block_till_done()
 
     sensor = hass.states.get("sensor.test")
-    assert sensor.state == "30"
     assert sensor.attributes.get("unit_of_measurement") == TIME_MINUTES
-    assert sensor.attributes.get(ATTR_ATTRIBUTION) is None
+    assert (
+        sensor.attributes.get(ATTR_ATTRIBUTION)
+        == "With the support of HERE Technologies. All information is provided without warranty of any kind."
+    )
+    if traffic_mode:
```

Comment:
It would be better to parametrize the expected sensor state instead of traffic mode so we don't need to include conditional logic in the tests.